karmagrambot/commands.py

```diff
@@ -51,9 +59,12 @@ def karma(_: Bot, update: Update):
         message.reply_text(f'Could not find user named {username}')
         return
 
-    user_karma = analytics.get_karma(user_info.user_id, message.chat_id)
+    user_karma = analytics.get_karma(user_info.user_id, message.chat_id, period)
 
-    message.reply_text(f'{user_info.username} has {user_karma} karma in this chat')
+    reply_msg = f'{user_info.username} has {user_karma} karma in this chat'
```

Comment:
How about:

```python
period_suffix = f'(since {period})' if period is not None else '(all time)'.

message.reply_text(f'{user_info.username} has {user_karma} karma in this chat {period_suffix}.')
```